packages/services/api/src/modules/integrations/providers/github-integration-manager.ts

```diff
@@ -71,10 +73,27 @@ export class GitHubIntegrationManager {
       },
     });
     this.logger.debug('Updating organization');
-    await this.storage.addGitHubIntegration({
+    const result = await this.storage.addGitHubIntegration({
       organizationId: input.organizationId,
       installationId: input.installationId,
     });
+
+    const currentUser = await this.session.getViewer();
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_UPDATED_INTEGRATION',
+      integrationId: input.installationId,
+      updatedFields: JSON.stringify({
+        gitHubIntegrationStatus: 'ENABLED',
+      }),
```

Comment:
TODO @TuvalSimha, I don't think we need `updatedFields` here if only the status would change.